examples/segmentation_3d/unet_evaluation_dict.py

```diff
@@ -62,24 +63,30 @@ def main():
         val_ds, batch_size=1, num_workers=4, collate_fn=list_data_collate, pin_memory=torch.cuda.is_available()
     )
 
-    device = torch.device("cuda:0")
+    # try to use all the available GPUs
+    devices = get_devices_spec(None)
     model = UNet(
         dimensions=3,
         in_channels=1,
         out_channels=1,
         channels=(16, 32, 64, 128, 256),
         strides=(2, 2, 2, 2),
         num_res_units=2,
-    ).to(device)
+    ).to(devices[0])
 
     model.load_state_dict(torch.load("best_metric_model.pth"))
+
+    # if we have multiple GPUs, set data parallel to execute sliding window inference
+    if len(devices) > 1:
+        model = torch.nn.DataParallel(model, device_ids=devices)
+
     model.eval()
     with torch.no_grad():
         metric_sum = 0.0
         metric_count = 0
         saver = NiftiSaver(output_dir="./output")
         for val_data in val_loader:
-            val_images, val_labels = val_data["img"].to(device), val_data["seg"].to(device)
+            val_images, val_labels = val_data["img"].to(devices[0]), val_data["seg"].to(devices[0])
```

Comment:
will `.to(devices[0], non_blocking=True)` make it faster?

Reply:
Hi @wyli ,

Thanks for your review.
According to https://discuss.pytorch.org/t/should-we-set-non-blocking-to-true/38234/4, I think here we have logic to save output into files, so `non_blocking=True` will not benefit performance.
And to keep this example straight-forward to show just multi-GPU inference, I suggest using the default value.
What do you think?

Thanks.